frontend/src/components/experimentRuns/ExperimentRunValidation.tsx

```diff
@@ -115,7 +115,10 @@ function ExperimentRunValidation({ experimentRunName }: ExperimentRunValidationP
 						<Collapse.Panel 
 							key={createLaneKey(lane)}
 							header={<Title level={5}>{`Lane ${lane.laneNumber}`}</Title>}
-							extra={<LaneValidationStatus validationStatus={lane.validationStatus} isValidationInProgress={isValidationInProgress}/>}
+							extra={<Space direction={'horizontal'}>
+								<LaneValidationStatus validationStatus={lane.validationStatus} isValidationInProgress={isValidationInProgress}/>
+								{lane.validationTime ? ['-', `${new Date(lane.validationTime).toLocaleDateString()}`] : ''}
```

Comment:
Date format is not correct ... use "fr-CA" for the locale ...

Reply:
I see in other places "en-US" is used instead ;)
- `frontend/src/components/experimentRuns/ExternalExperimentRunDetailContent.tsx:73`
- frontend/src/utils/humanReadableTime.js:17

But maybe they are not used.

Reply:
some locale may have format that are useful... I want uniformity in the date format the other times / human readable times i am not really that much bothered.